File: qoshimcha/task09.js
let num1 = 11
let num2 = 12
let num3 = 10
if (num1 == num2 && num2 == num3) {
    console.log('Teng tomonli');
}else if (num1 == num2 || num2 == num3 || num1 == num3) {
    console.log('Teng yonli');
}else{
    console.log('Turli tomonli');
}

File: task09.py
num1 = int(input('uchburchak tomonlari1: '))
num2 = int(input('uchburchak tomonlari2: '))
num3 = int(input('uchburchak tomonlari3: '))
if num1 == num2 and num2 == num3:
    print('Teng tomonli')
else:
    if num1 == num2 or num2 == num3 or num1 == num3:
        print('Teng yonli')
    else:
        print('Turli tomonli')
